lib/resources/r4/patient.yaml

```diff
@@ -209,6 +209,7 @@ fields:
       <li> Up to two `given` names may be provided. If additional given names are required they must be provided in the second value as a single string separated by blank spaces. </li>
       <li> Only one `prefix` and one `suffix` may be provided. </li>
       <li> When specifying a `period`, the fields must include a time component with a timezone. </li>
+      <li> `family`, `given`, `prefix`, and `suffix` have a character limit of 100. </li> 
```

Comment:
for clarity on arrays we should finagle this a bit

```
* The `family` field has a character limit of 100.
* Each value for the `given`, `prefix`, and `suffix` fields has a character limit of 100.
```

applies to address.line as well

Reply:
https://github.com/cerner/fhir.cerner.com/pull/414/commits/5a9a9c3bf70e3347b6761980185cdc7219853baa#diff-07c686ec4248977e084ba5a602c761ceR213
https://github.com/cerner/fhir.cerner.com/pull/414/commits/5a9a9c3bf70e3347b6761980185cdc7219853baa#diff-07c686ec4248977e084ba5a602c761ceR295